base/initdefs.jl

```diff
@@ -40,8 +40,13 @@ function init_load_path()
     if haskey(ENV, "JULIA_LOAD_PATH")
         prepend!(LOAD_PATH, split(ENV["JULIA_LOAD_PATH"], @static is_windows() ? ';' : ':'))
     end
-    push!(LOAD_PATH, abspath(JULIA_HOME, "..", "local", "share", "julia", "site", vers))
-    push!(LOAD_PATH, abspath(JULIA_HOME, "..", "share", "julia", "site", vers))
+    if Base.DARWIN_FRAMEWORK 
+        push!(LOAD_PATH, abspath(homedir(),"Library",Base.DARWIN_FRAMEWORK_NAME,vers,"site"))
+        push!(LOAD_PATH, abspath("/Library",Base.DARWIN_FRAMEWORK_NAME,vers,"site"))
```

Comment:
Right, so I would do something like Library/Julia/site/VERSDIR instead then?  Or are you suggesting stick to prefix/share/julia as the base directory structure over the os x standard of [~]/Library/Julia?